<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessDialog</class>
 <widget class="QDialog" name="ProcessDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="ImageLabel" name="exitLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>360</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber1">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>20</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="digitCount">
    <number>2</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber2">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>20</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="digitCount">
    <number>2</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber3">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>20</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="digitCount">
    <number>2</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QLabel" name="stepImage_1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="stepImage_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="stepImage_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="stepImage_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="stepImage_5">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="stepImage_6">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>130</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="stepImage_7">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>130</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="stepImage_8">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>130</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="stepImage_9">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>130</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="stepLabel_1">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>100</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>第1步</string>
   </property>
  </widget>
  <widget class="QLabel" name="stepLabel_2">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>100</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>第2步</string>
   </property>
  </widget>
  <widget class="QLabel" name="stepLabel_3">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>100</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>第3步</string>
   </property>
  </widget>
  <widget class="QLabel" name="stepLabel_4">
   <property name="geometry">
    <rect>
     <x>305</x>
     <y>100</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>第4步</string>
   </property>
  </widget>
  <widget class="QLabel" name="stepLabel_5">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>100</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>第5步</string>
   </property>
  </widget>
  <widget class="QLabel" name="stepLabel_6">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>100</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>第6步</string>
   </property>
  </widget>
  <widget class="QLabel" name="stepLabel_7">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>100</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>第7步</string>
   </property>
  </widget>
  <widget class="QLabel" name="stepLabel_8">
   <property name="geometry">
    <rect>
     <x>585</x>
     <y>100</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>第8步</string>
   </property>
  </widget>
  <widget class="QLabel" name="stepLabel_9">
   <property name="geometry">
    <rect>
     <x>655</x>
     <y>100</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>第9步</string>
   </property>
  </widget>
  <widget class="Line" name="line_1">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>141</y>
     <width>46</width>
     <height>3</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>204</x>
     <y>141</y>
     <width>46</width>
     <height>3</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>274</x>
     <y>141</y>
     <width>46</width>
     <height>3</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>344</x>
     <y>141</y>
     <width>46</width>
     <height>3</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>414</x>
     <y>141</y>
     <width>46</width>
     <height>3</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>484</x>
     <y>141</y>
     <width>46</width>
     <height>3</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>554</x>
     <y>141</y>
     <width>46</width>
     <height>3</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>624</x>
     <y>141</y>
     <width>46</width>
     <height>3</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="picture">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>200</y>
     <width>170</width>
     <height>197</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ImageLabel" name="outLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>190</y>
     <width>142</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ImageLabel" name="inLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>250</y>
     <width>142</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ImageLabel" name="startLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>340</y>
     <width>142</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ImageLabel" name="cancelLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>400</y>
     <width>142</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="dateLabel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageLabel</class>
   <extends>QLabel</extends>
   <header>component/imagelabel.h</header>
   <slots>
    <signal>released()</signal>
    <signal>pressed()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
